msmtp/1.8.7/wolfssl-msmtp-1.8.7.patch

```diff
@@ -0,0 +1,377 @@
+diff --git a/configure.ac b/configure.ac
+index 63e47aa..43965b0 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -98,25 +98,34 @@ tls_lib="none"
+ want_tls="yes"
+ want_gnutls="yes"
+ want_openssl="no"
++want_wolfssl="no"
+ tls_CFLAGS=""
+ tls_LIBS=""
+ with_ssl_was_used=no
+-AC_ARG_WITH([tls], [AS_HELP_STRING([--with-tls=[gnutls|openssl|no]],
+-    [TLS support: GnuTLS (default), OpenSSL (discouraged), or none.])],
++AC_ARG_WITH([tls], [AS_HELP_STRING([--with-tls=[gnutls|openssl|wolfssl|no]],
++    [TLS support: GnuTLS (default), OpenSSL (discouraged), wolfSSL, or none.])],
+     if test "$withval" = "gnutls"; then
+ 	want_tls=yes
+ 	want_gnutls=yes
+ 	want_openssl=no
++    want_wolfssl=no
+     elif test "$withval" = "openssl"; then
+ 	want_tls=yes
+ 	want_gnutls=no
+ 	want_openssl=yes
++    want_wolfssl=no
++    elif test "$withval" = "wolfssl"; then
++    want_tls=yes
++    want_gnutls=no
++    want_openssl=no
++    want_wolfssl=yes
+     elif test "$withval" = "no"; then
+ 	want_tls=no
+ 	want_gnutls=no
+ 	want_openssl=no
++    want_wolfssl=no
+     else
+-	AC_MSG_ERROR([Use --with-tls=gnutls or --with-tls=openssl or --with-tls=no])
++	AC_MSG_ERROR([Use --with-tls=gnutls or --with-tls=openssl or --with-tls=wolfssl or --with-tls=no])
+     fi)
+ if test "$want_gnutls" = "yes"; then
+     PKG_CHECK_MODULES([libgnutls], [gnutls >= 0.0], [HAVE_LIBGNUTLS=1], [HAVE_LIBGNUTLS=0])
+@@ -147,6 +156,21 @@ if test "$want_openssl" = "yes" -a "$have_tls" = "no"; then
+ 	AC_DEFINE([HAVE_LIBSSL], [1], [Define to 1 if libssl is available])
+     fi
+ fi
++if test "$want_wolfssl" = "yes" -a "$have_tls" = "no"; then
++    AC_MSG_WARN([Using OpenSSL is discouraged; consider using GnuTLS instead])
```

Comment:
Is this AC_MSG_WARN accurate?  Looks like this is in the wolfSSL section, but mentions OpenSSL and GnuTLS.

Reply:
Nice catch. I appear to have left this in by accident after copy-pasting from the OpenSSL stanza.